.github/workflows/validate-kind.yaml

```diff
@@ -0,0 +1,160 @@
+name: validate-kind-e2e
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+      - reopened
+    branches:
+      - 'renovate/**'
+    paths:
+    - 'clusters/kind-cluster/**'
+
+jobs:
+  kubernetes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@main
+      - name: Init Homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      - name: Install Flux
+        run: |
+          brew install fluxcd/tap/flux@2.3
+          flux -v
+      - name: Init Kind action
+        uses: helm/kind-action@v1.12.0
+        with:
+          cluster_name: sqnc-flux-infra
+          config: ./scripts/kind-cluster-config.yaml
+      - name: Install Flux to cluster
+        run: flux install
+      - name: Configure Git
+        run: |
+          git config user.name ${GITHUB_ACTOR}
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+      - name: Create source and kustomization on main
+        if: success()
+        run: |
+          flux create source git flux-system \
+          --url=${{ github.event.repository.html_url }} \
+          --branch=main \
+          --username=${GITHUB_ACTOR} \
+          --password=${{ secrets.GITHUB_TOKEN }}
+          flux create kustomization flux-system \
+          --source=flux-system \
+          --path=./clusters/kind-cluster/base
+      - name: Verify Flux kustomizations
+        run: |
+          kubectl -n kube-system wait kustomization/nginx-sync --for=condition=ready --timeout=10m
+          kubectl -n keycloak wait kustomization/keycloak-sync --for=condition=ready --timeout=10m
+          kubectl -n monitoring wait kustomization/monitoring-sync --for=condition=ready --timeout=10m
+          kubectl -n loki wait kustomization/promtail-sync --for=condition=ready --timeout=10m
+          kubectl -n loki wait kustomization/loki-sync --for=condition=ready --timeout=10m
+          kubectl -n alice wait kustomization/alice-sync --for=condition=ready --timeout=10m
+          kubectl -n bob wait kustomization/bob-sync --for=condition=ready --timeout=10m
+          kubectl -n charlie wait kustomization/charlie-sync --for=condition=ready --timeout=10m
```

Comment:
This can be replaced with `kubectl wait -A kustomization -l kustomize.toolkit.fluxcd.io/name=flux-system --for=condition=ready` Unless there is a reason we are doing this individually?